add option to read append-data from a file

solidgoldbomb opened the pull request.

iagox86 commented:
Sorry, it's like 4 years late, but this looks like a great feature and - annoyingly - it's conflicting now. If you fix this I'll happily merge it! Thanks!

solidgoldbomb commented:
On closer inspection, maybe just a random coincidence but it appears as though someone else may have reworked my branch and pushed their own MR (# 17) that included my changes under their own authorship.  That other MR pushed 11 days after mine has already been merged and includes nearly identical code to my MR, even down to the "amd" -> "and" typo that I also fixed here.  They also added support for output to a file which isn't included here so that was helpful.

That explains the conflict.  Either way, it looks like this MR can be closed.

iagox86 commented:
Okay cool! Sorry about the delay, I really don't pay enough attention to my PR queues
